### DISCUSSION TITLE
Maybe the extension dependencies resolving need to invoke some `install.py`

### DISCUSSION THREAD

**Discussion by SuperMarioSF** [AbdBarho/stable-diffusion-webui-docker#523] [Ideas]
For some reason, sd-dynamic-prompt extension doesn't have a proper `requirements.txt` provided, instead it provided a `install.py`, and this file will be invoked by auto's sd-webui when installing the extension.

I guess it is a good idea to at least run this script once to get the dependencies done. So a search and invoking `install.py` may needed in `entrypoint.sh`.

ref: issue #400

**Comment by AbdBarho** [comment]
This should be already fixed in https://github.com/AbdBarho/stable-diffusion-webui-docker/commit/6365811f3552766b5ed8f1b459a97d397cf9f7e0

is it not the case?

**Reply by SuperMarioSF** [reply]
Sorry for this very late reply.
I recently reinstalled an instance and everything works fine now, even for updates. The issue is no more.

Thanks.
